content/src/content/jcr_root/apps/core/wcm/components/modal/v1/modal/clientlibs/site/css/modal.css

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright 2019 Adobe Systems Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+.tingle-modal-box__content {
+    max-height: 80vh;
```

Comment:
What is the reason for this change in the default look and feel / scroll behaviour? It seems to cause layout problems in the responsive behaviour that comes shipped with tingle.

<img width="525" alt="Screenshot 2019-04-24 at 10 10 31" src="https://user-images.githubusercontent.com/25616660/56647498-7914e800-6679-11e9-9aaa-2207ed8804a5.png">